server.go

```diff
@@ -8,18 +8,69 @@ import (
 
 	"github.com/tus/tusd/pkg/filestore"
 	tusd "github.com/tus/tusd/pkg/handler"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/jwtauth"
+
+	_ "github.com/joho/godotenv/autoload"
 )
 
 var dir = flag.String("dir", "tusd-files", "where the uploaded files should be stored")
 
+var tokenAuth *jwtauth.JWTAuth
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello")
+}
+
+func init() {
+	secret := os.Getenv("JWTAuthSecret")
+	tokenAuth = jwtauth.New("HS256", []byte(secret), nil)
+	// For debugging/example purposes, we generate and print
+	// a sample jwt token with claims `user_id:123` here:
+	// Don't forget to import jwt "github.com/dgrijalva/jwt-go"
+	// _, tokenString, _ := tokenAuth.Encode(jwt.MapClaims{"user_id": 123})
+	// fmt.Printf("DEBUG: a sample jwt is %s\n\n", tokenString) */
+}
+
 func main() {
 	flag.Parse()
 
 	_, err := os.Stat(*dir)
 	if os.IsNotExist(err) {
 		err = os.MkdirAll(*dir, 0777)
 	}
+	err = http.ListenAndServe(":8080", router())
```

Comment:
```suggestion
	err = http.ListenAndServe(":8000", router())
```

`8080` is already used by Vue to serve the client during development